unblob/pool.py

```diff
@@ -85,18 +115,31 @@ def close(self):
     def _clear_input_queue(self):
         try:
             while True:
+                logger.warning("getnowait")
```

Comment:
Is this really a warning case? It would be printed once even if the queue was empty and we are in a normal shutdown, wouldn't it?

Reply:
Wow, sorry, these new warnings were just were some debug messages for me during development, which somehow slip through and remained here. Sorry, I will remove them.